pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -231,6 +261,8 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: AdminRoleName,
+				Annotations: map[string]string{
+					oapi.OpenShiftDescription: "A project manager. If used in a local binding, an admin user will have rights to view any resource in the project and modify any resource in the project except for role creation and quota. If the cluster-admin wants to allow an admin to modify roles, the cluster-admin must create a project-scoped Policy object."},
```

Comment:
I don't think we want these getting too technical, these descriptions are for end users deciding why they want to choose one role vs another.  What if we simplify this whole thing down to:

"A user that has edit rights within the project and can change the project's membership."

Maybe contentious since membership is a console term?